src/components/GroupSuggestions/index.js

```diff
@@ -1,26 +1,70 @@
-import React, { memo } from 'react'
+import React, { memo, useState } from 'react'
 import './styles.scss'
 import propTypes from 'prop-types'
+import Swipeable from 'react-swipy'
 import SuggestionBox from './SuggestionBox'
+import ButtonContainer from '../ButtonContainer'
 
 const GroupSuggestions = props => {
-  const { channels, handleJoinChannel, getMembersByChannelId } = props
+  const { channels, handleJoinChannel, channelMembers } = props
+
+  const [cards, setCards] = useState(channels)
+  const remove = () => setCards(cards.slice(0, -1))
+
   return (
     <div className="group-suggestions">
       <h1>Ehdotetut ryhmät</h1>
       {!channels && (
         <div>Saat lisää ehdotuksia vastaamalla päivän kysymyksiin.</div>
       )}
       <div className="group-suggestion-boxes">
-        {channels &&
-          channels.map(channel => (
-            <SuggestionBox
-              key={channel.id}
-              channel={channel}
-              handleJoinChannel={handleJoinChannel}
-              getMembersByChannelId={getMembersByChannelId}
-            />
-          ))}
+        {cards && cards.length > 0 && (
+          <div>
+            {cards &&
+              cards.map((card, i) =>
+                i === cards.length - 1 ? (
+                  <Swipeable
+                    key={card.id}
+                    min={80}
+                    buttons={({ left }) => (
+                      <div className="suggestion-buttons-wrapper">
+                        <ButtonContainer
+                          className="skip-suggestion-button"
+                          onClick={() => {
+                            left()
+                          }}
+                        >
+                          Ohita
+                        </ButtonContainer>
+                        <ButtonContainer
+                          onClick={handleJoinChannel(card.id)}
+                          className="join-suggestion-button"
+                          secondary
+                        >
+                          Liity
+                        </ButtonContainer>
```

Comment:
On a bigger screen these buttons are really far apart, which feels a bit heavy to use. Although maybe we don't need  to worry about bigger screens so much.

Reply:
Good point! I would say it's still ok since the entire app has max width, and will never be wider than that.